docs/source/notebooks/AR.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<ul><li><code>%matplotlib inline</code> is no longer needed in NBs</li><li>Can reorganise imports alphabetically:</li></ul><pre class="ql-syntax" spellcheck="false">import arviz as az
import matplotlib.pyplot as plt
import pymc3 as pm
import numpy as np
</pre><ul><li>Can you set the style in a cell below this one (this is because of matplotlib sets the default styles) and use ArviZ style instead, which is the newest way:</li></ul><pre class="ql-syntax" spellcheck="false">az.style.use('arviz-darkgrid')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>